src/i18n/en/nav.ts

```diff
@@ -46,56 +46,52 @@ export default [
 	},
 	{ text: 'Pages', slug: 'core-concepts/astro-pages', key: 'core-concepts/astro-pages' },
 	{ text: 'Layouts', slug: 'core-concepts/layouts', key: 'core-concepts/layouts' },
-
-	{ text: 'Recipes', header: true, type: 'learn', key: 'examples' },
-	{ text: 'Migrate to Astro', slug: 'guides/migrate-to-astro', key: 'guides/migrate-to-astro' },
-	{ text: 'Connect a CMS', slug: 'guides/cms', key: 'guides/cms' },
-	{ text: 'Add backend services', slug: 'guides/backend', key: 'guides/backend' },
-	{ text: 'Add integrations', slug: 'guides/integrations-guide', key: 'guides/integrations-guide' },
-	{ text: 'Deploy your site', slug: 'guides/deploy', key: 'guides/deploy' },
-	{ text: 'More recipes', slug: 'recipes', key: 'guides/recipes' },
-
-	{ text: 'Guides', header: true, type: 'learn', key: 'features' },
 	{
 		text: 'Astro Template Syntax',
 		slug: 'core-concepts/astro-syntax',
 		key: 'core-concepts/astro-syntax',
 	},
-	{
-		text: 'UI Frameworks',
-		slug: 'core-concepts/framework-components',
-		key: 'core-concepts/framework-components',
-	},
-	{ text: 'Routing', slug: 'core-concepts/routing', key: 'core-concepts/routing' },
-	{ text: 'Markdown & MDX', slug: 'guides/markdown-content', key: 'guides/markdown-content' },
+
+	{ text: 'Built-ins', header: true, type: 'learn', key: 'examples' },
```

Comment:
Is `key: 'examples'` correct? I don't totally grok what `key` is for to be honest!

Reply:
Good old copy paste! :sweat_smile: